Batch fertilizer concentrations

KrishnaTO opened the pull request:
Using dosdp-tools, created pattern file (CAD) to create chemical-specific `chemical area density` linking list of chemicals (associated with CHEBI's fertilizer [role]) and area densities

Additional details and closes issue #122 

@marieALaporte 
@celineaubert

KrishnaTO committed:
make new patterns

chemical_area_density pattern

@marieALaporte

marieALaporte committed:
Update chemical_area_density.yaml

should be working now.

KrishnaTO committed:
created pattern file and added parent classes in csv

added fertilizer_chemicals-area_density as subclasses to molecular_entity-area_density

KrishnaTO committed:
Updated yaml pattern

connect CAD to chemicals under respective area densities

KrishnaTO committed:
Merge branch 'master' into batch-fertilizer-concentrations

KrishnaTO committed:
Merge branch 'batch-fertilizer-concentrations' into patch-1

KrishnaTO committed:
Merge pull request #2 from marieALaporte/patch-1

Update chemical_area_density.yaml

KrishnaTO committed:
Added Inorganic fertilizer role per issue #95

marieALaporte commented:
could you update the pattern file, so the categories can be inferred and not asserted, as a fertilizer can belong to multiple categories?
here an example for 'calcium area density': `'area density' and ('inheres in' some (has_part some 'calcium atom'))`

KrishnaTO committed:
Revise CC pattern and undo agro-edit changes

KrishnaTO committed:
Update cad patterns yaml & update make recipe

updated equivalent classes
updated make/patterns recipe to include robot merge

KrishnaTO committed:
Checkpoint-Agro-edit.owl - pre-merge

KrishnaTO committed:
Added patterns for inorganic fertilizers w/ eq statements

Created chemical_area_density_eq: added eq statements to existing inorganic fertiliser classes via pattern
Updated chemical_area_density: added eq statements

KrishnaTO committed:
Updated with eq for terms

KrishnaTO commented:
@marieALaporte Done! can be merged.

see latest 2 commit for patterns added.

marieALaporte committed:
Update chemical_area_density_eq.tsv

marieALaporte committed:
make area density eq patterns

marieALaporte committed:
removing old equivalent axioms
